src/Sarif.UnitTests/Converters/AndroidStudioConverterTests.cs

```diff
@@ -345,8 +363,14 @@ public void AndroidStudioConverter_ConvertSarifResult_GeneratesLocationWithOnlyP
 
             LocationInfo locationInfo = GetLocationInfoForBuilder(builder);
 
-            locationInfo.Location.Should().Be(expectedLocation);
-            locationInfo.LogicalLocationComponents.SequenceEqual(expectedLogicalLocationComponents).Should().BeTrue();
+            LocationEqualityComparer.Instance.Equals(locationInfo.Location, expectedLocation)
+                .Should().BeTrue();
+
+            locationInfo.LogicalLocationComponents
+                .SequenceEqual(
+                    expectedLogicalLocationComponents,
+                    LogicalLocationComponentEqualityComparer.Instance)
```

Comment:
Huh, actually `ValueEquals` is a better name, and if you type `foo.Eq` it pops up in the Intellisense right next to `Equals`. What do you think?